content/webapp/pages/articles.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import {Component} from 'react';
+import {getArticles} from '@weco/common/services/prismic/articles';
+import {convertImageUri} from '@weco/common/utils/convert-image-uri';
+import {articleLd} from '@weco/common/utils/json-ld';
+import PageWrapper from '@weco/common/views/components/PageWrapper/PageWrapper';
+import LayoutPaginatedResults from '@weco/common/views/components/LayoutPaginatedResults/LayoutPaginatedResults';
+import type {GetInitialPropsProps} from '@weco/common/views/components/PageWrapper/PageWrapper';
+import type {Article} from '@weco/common/model/articles';
+import type {PaginatedResults} from '@weco/common/services/prismic/types';
+
+type Props = {|
+  articles: PaginatedResults<Article>
+|}
+
+const pageDescription = 'Our words and pictures explore the connections between science, medicine, life and art. Dive into one no matter where in the world you are.';
```

Comment:
The second sentence doesn't make sense – I imagine the first one used to say something like 'stories' instead of 'words and pictures' at some point? Ping @jennpb

Reply:
Was the change made on Jen T-S's request? There may be a reason for it to be written that way, so we'd best check with her.

Reply:
![screen shot 2018-10-24 at 17 11 39](https://user-images.githubusercontent.com/31692/47445072-e4a3f880-d7af-11e8-92cd-708a442b8bcc.png)

Reply:
Which points out the fact that that title isn't in the new version... will get on it.

Reply:
<img width="546" alt="screen shot 2018-10-24 at 17 49 16" src="https://user-images.githubusercontent.com/1394592/47447379-42870f00-d7b5-11e8-99a7-a283f352cedd.png">

Reply:
Want to do it in this PR? Or I'll make a ticket.

Reply:
Ticket and test it?